README.md

```diff
@@ -28,3 +28,8 @@ Start sidekiq from the root of your Rails application so the jobs will be proces
 bundle exec sidekiq
 ```
 
+## Importing from purl:
```

Comment:
@hannahfrost said just do Purl import for now, so the mods importer is left undocumented.  The mods that stanford has doesn't have links to collections in it.